secretmanager/create_secret_with_annotations.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package secretmanager
+
+// [START secretmanager_create_secret_with_annotations]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	"cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+)
+
+// createSecretWithAnnotations creates a new secret with the given name and annotations.
+func createSecretWithAnnotations(w io.Writer, parent, id string) error {
+	// parent := "projects/my-project"
+	// id := "my-secret"
+
+	annotationKey := "annotationkey"
+	annotationValue := "annotationvalue"
+
+	// Create the client.
```

Comment:
issue: remove this comment. It doesn't add anything to the sample.

See https://googlecloudplatform.github.io/samples-style-guide/#comments